### DISCUSSION TITLE
Blacklist server in-game should default by-endpoint instead of by-name.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15171] [Ideas and suggestions]
Blacklisting a server using the in-game context menu only adds it's name to the blacklist, which is kind of worthless since the same host with a different named server bypasses it entirely, and also that can create lots of false-positives if the server name is something really common like "outpost" or "campaign".

It makes alot more functional sense for the default behavior to be blacklist by endpoint, 
because the server IP is really the most 'identifying' aspect of a server.

**Comment by AziKarnage** [comment]
Vidya games I wanna find you on discord, I am azikarnage
